### DISCUSSION TITLE
403 error on startup

### DISCUSSION THREAD

**Discussion by BenPanbo** [jertel/vuegraf#79] [General]
Hi, 

I've been using vuegraf for a while but haven't used it in a bit. I noticed that I'm getting a 403 error when it starts up. I figured this was due to API changes so I updated vuegraf and pyemvue but I'm still getting errors. Here's what's logged in the execution log:

Fatal error: (<class 'requests.exceptions.HTTPError'>, HTTPError('403 Client Error: Forbidden for url: https://api.emporiaenergy.com/customers/6452/devices?detailed=true&hierarchy=true'), <traceback object at 0xb4aab620>

Any suggestions?

Thanks!
Ben

**Comment by jertel** [comment]
#14 looks similar to your discussion. Make sure you're correctly updating both Vuegraf and PyEmVue.
